### PR TITLE
#3 Fixed

### LINKED ISSUE #3: Python scripts should be runnable in python 3
At least `print` syntax is invalid.

### PR DESCRIPTION
**Pull request by mfbehrens99**
#3 Scripts are now also available in Python 3

### CODE REVIEW

**Comment by Zverik** [comment]
Thank you for the PR! Since python2 is EOL, could you please just replace python2 with python3, to not confuse users?

**Comment by mfbehrens99** [comment]
It's done!

File: www/python.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # Generated by BigMap 2. Permalink: <?=$permalink ?>
 
 
-import io, urllib2, datetime, time, re, random
+import io, urllib.request, urllib.error, urllib.parse, datetime, time, re, random
```
Comment (Zverik) [inline www/python.php:16]: You're not using `.error` and `.parse`, do you need to import these?

File: www/python.php [hunk 2]
```diff
@@ -32,24 +33,24 @@
...
 
 draw = ImageDraw.Draw(resultImage)
-draw.text((5, ysize*256-15), attribution, (0,0,0))
+draw.text((5, ysize*tilesize-15), attribution, (0,0,0))
 del draw
 
 now = datetime.datetime.now()
 outputFileName = "map%02d-%02d%02d%02d-%02d%02d.png" % (zoom, now.year % 100, now.month, now.day, now.hour, now.minute)
-resultImage.save(outputFileName)
+resultImage.save(outputFileName)
```
Comment (Zverik) [inline www/python.php:56]: Please restore the end-of-file newline :)

**Pull request merged**